### DISCUSSION TITLE
Inquiries about the MONAI translation

### DISCUSSION THREAD

**Discussion by ghost** [Project-MONAI/MONAI#3218] [Q&A]
Hello, I am a Korean university student interested in artificial intelligence open source.

I think it would be better for Koreans to use MONAI more easily, so I want to make a Korean translation of GitHub documents.
So, I'd like to ask if I can upload a translation to my GitHub account and leave a source on the MONAI GitHub site.

I look forward to hearing from you. 
Thank you.

**Comment by Nic-Ma** [comment]
Hi @seyeon2 ,

Thanks for your interest here.
I think that's an interesting idea, @wyli @ericspod maybe we should raise this topic to some team discussion first?

Thanks.

**Reply by ghost** [reply]
Hi @Nic-Ma ,
Thank you for your answer.

I'm not going to use it for commercial purposes, but for academic purposes.
Please let me know the results after the team discussion.

I look forward to hearing from you.
Thank you.

**Reply by ericspod** [reply]
I think we definitely should do some discussion on the broader topic of documentation translation.

**Reply by ghost** [reply]
Hello, @Nic-Ma, @ericspod. 

In order to help Korean users understand, we translated README.md of the Pinned six repositories. 
The link in progress is as follows.
https://github.com/april6607/MONAI_translation

After completion, I would like to ask if we can request a 'full-request' for the documents we have proceeded.

I look forward to hearing from you.
Thank you.

**Reply by wyli** [reply]
Is the plan to cover all the docs in https://github.com/Project-MONAI/MONAI/tree/dev/docs? in that case perhaps we should consider localization via readthedocs? https://docs.readthedocs.io/en/stable/localization.html (currently https://docs.monai.io/en/latest/ is hosted on readthedocs https://readthedocs.org/projects/monai/)
